example/deployments/cumulus/sandbox.tfvars

```diff
@@ -46,4 +46,6 @@ ems_retention_in_days = 30
 ems_submit_report     = true
 ems_username          = "cumulus"
 
+es_request_concurrency = 1
```

Comment:
So the whole point of this is to make the dbIndexer run its requests in serial, rather than parallel, by setting the concurrency to 1? If so, I'd rather us just explicitly remove the parallel processing via Promise.all from dbIndexer. Basically, if it's the concurrency causing an issue that our tests are revealing, I don't think it should even be possible to enable concurrency in that lambda at least.

Reply:
Ok.  I will remove the concurrency code from dbIndexer.  We can support concurrency for indexing records from different types. But that would be separate ticket.
15a9dca[https://github.com/nasa/cumulus/pull/1970/commits/15a9dca3b889a14c80b2a2ee7deceb225c589114]